src/components/Picker/index.js

```diff
@@ -76,4 +97,5 @@ class Picker extends PureComponent {
 Picker.propTypes = propTypes;
 Picker.defaultProps = defaultProps;
 
-export default Picker;
+// eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
I saw other files disabling spreading props lint error with this comment. If this file is the exception I will work on make the changes.